accounts/models.py

```diff
@@ -260,3 +260,17 @@ def create_user_profile(sender, instance, created, **kwargs):
         profile.save()
 
 post_save.connect(create_user_profile, sender=User)
+
+class DonationCampaign(models.Model):
+    goal = models.DecimalField(max_digits=6, decimal_places=2)
+    date_start = models.DateField()
+
+
+class Donation(models.Model):
+    email = models.CharField(max_length=255)
```

Comment:
I agree with @bdejong 's point. Donations could work same way as for contact form.